pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/store/SegmentDirectory.java

```diff
@@ -140,6 +141,14 @@ public void acquire(FetchContext fetchContext) {
   public void release(FetchContext fetchContext) {
   }
 
+  /**
+   * Copy segment directory to a local directory.
+   * @param dest the destination directory
+   */
+  public void copyTo(File dest)
```

Comment:
`copyTo` -> `copyToLocal`?

Reply:
I'd prefer copyTo as it doesn't imply where the source is. copyToLoca() kinda implies the source is remote.